bin/zeppelin-daemon.sh

```diff
@@ -67,6 +67,12 @@ if [[ -d "${ZEPPELIN_HOME}/zeppelin-server/target/classes" ]]; then
   ZEPPELIN_CLASSPATH+=":${ZEPPELIN_HOME}/zeppelin-server/target/classes"
 fi
 
+# Add jdbc connector jar
```

Comment:
This looks an example or tutorial, thus could you please remove the below and update docs?